src/components/primitives/Checkbox/CheckboxGroup.tsx

```diff
@@ -1,42 +1,40 @@
-import React from 'react';
+import React, { createContext } from 'react';
+import { useCheckboxGroupState } from '@react-stately/checkbox';
+import { useCheckboxGroup } from '@react-native-aria/checkbox';
 import {
   FormControlContext,
   IFormControlContext,
 } from '../../composites/FormControl';
+import type { ICheckboxGroupProps, ICheckboxContext } from './types';
 import Box from '../Box';
-import type { ICheckboxGroupProps } from './types';
-import { useCheckboxGroup } from './useCheckboxGroup';
 
-export const CheckboxContext = React.createContext({});
+export let CheckboxGroupContext = createContext<ICheckboxContext | null>(null);
 
-const CheckboxGroup = ({
+export function CheckboxGroup({
```

Comment:
ChecboboxGroup is also the default export in this file. Please remove this export